src/Core/src/Handlers/Element/ElementHandlerAttribute.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+
+namespace Microsoft.Maui.Handlers;
+
+internal abstract class ElementHandlerAttribute : Attribute
+{
+	public abstract IElementHandler CreateHandler();
+	public abstract Type HandlerType { get; }
+}
+
+internal sealed class ElementHandlerAttribute<THandler> : ElementHandlerAttribute
+	where THandler : IElementHandler, new()
+{
+	public override IElementHandler CreateHandler()
+	{
+		var handler = new THandler();
+		RemappingHelper.RemapIfNeeded(typeof(THandler), handler);
```

Comment:
I realized this doesn't work. It's the _element_ that can be remapped, not the `handler`. I need to figure out how to address this without having to change the API drastically. The cctors approach might be the best after all.